urlview.py

```diff
@@ -32,8 +32,14 @@ def urlview(data, buf, args):
         weechat.prnt(buf, "No URLs found")
         return weechat.WEECHAT_RC_OK
 
+    if weechat.config_is_set_plugin("command"):
+        command = weechat.config_get_plugin("command")
+    else:
```

Comment:
Can you update this to set the default value if it isn't set, and then read the command from the same place so we can eliminate the duplication in the else?